usaspending_api/common/threaded_data_loader.py

```diff
@@ -252,7 +253,10 @@ def load_data_into_model(self, model_instance, fields, field_map, value_map, row
             # If our field is the 'long form' field, we need to get what it maps to
             # in the data so we can map the data properly
             elif model_field in LONG_TO_TERSE_LABELS:
-                source_field = LONG_TO_TERSE_LABELS[model_field]
+                if type(mod) == TransactionFPDS:
```

Comment:
I don't understand why this mapping is necessary. Which table has `primary_place_of_performance_country_name`?

Reply:
I couldn't find where this would be hit but put it in place in case this is ever reached where the model is TransactionFPDS just in case.

Reply:
If I understand it correctly, these mappers are for data loads (specifically from Broker -> USAspending) and I don't see that field name in either FPDS DB  table.

Reply:
Yeh, I couldn't find it anywhere either. Will remove changes around this.